index.md

```diff
@@ -17,7 +17,11 @@ Join #cri-o on IRC (freenode)
 
 ### Distribution Packaging
 
-* **Fedora**: Available on all supported Fedora versions 28, 29 and Rawhide. 
+* **Fedora**: Available on all supported Fedora versions.
+	* Fedora 30 and Rawhide
```

Comment:
do we need a "And later" type of thing here.  Otherwise when F31 comes out in a few weeks you'll have to remember to up date this.

Reply:
ooh lower maintenance burden? I like